src/Searchable.php

```diff
@@ -643,18 +671,26 @@ protected function getSearchableFieldValuesForRelation($fieldName, $params, $cla
      */
     protected function formatBoolean($fieldValue)
     {
-        return boolval($fieldValue) ? 'true' : 'false';
+        return boolval($fieldValue) ? true : false;
```

Comment:
`boolval()` always returns a `bool` no need to append the ternary (predates this work - granted)

Reply:
Fixed